cfgov/wellbeing/jinja2/wellbeing/home.html

```diff
@@ -12,35 +12,16 @@
     {{ super() }}
     <style>
         .m-hero {
-            background-color: #cce3f5;
+            background-color: #b4d2d1;
         }
         .m-hero_image {
-            background-image: url( '/f/images/parents_hero_760x364.original.png' );
+            background-image: url( '{{ static("img/fwb_hero_760x300.png") }}' );
             filter: progid:DXImageTransform.Microsoft.AlphaImageLoader(
-                src='/f/images/parents_hero_760x364.original.png',
-                sizingMethod='scale'
+                    src='{{ static("img/fwb_hero_760x300.png") }}',
+                    sizingMethod='scale'
             );
             padding-bottom: 41.4893617%;
         }
-        @media screen and ( min-width: 37.5625em ) {
```

Comment:
It was copied from typical Wagtail hero output when I originally put it in there. In the case of a hero like this one, though, where the small screen image is the same as the large screen image, it serves no purpose.